client/components/Item/Card/Card.vue

```diff
@@ -55,7 +55,7 @@
           />
         </div>
         <div
-          v-if="overlay && !$browser.isMobile()"
```

Comment:
Why? I guess you wanted to switch to vuetify breakpoint but forgot about it

Reply:
It's because using `$browser.isMobile()` here completely breaks refreshing with SSR on some pages.

Reply:
So maybe use $vuetify?

Reply:
No, because we don't want small desktop windows to hide the overlay. Same this as below, this had a different purpose.

I added possible suitable replacement in the latest commit, using `window.matchMedia('(pointer:fine)').matches`.